en/advanced_config/sensor_thermal_calibration.md

```diff
@@ -1,25 +1,31 @@
 # Thermal Calibration and Compensation
 
-PX4 contains functionality to calibrate and compensate rate gyro, accelerometer and barometric pressure sensors for the effect of changing sensor temperature on sensor bias.
+PX4 contains functionality to calibrate and compensate accelerometer, gyro, magnetometer, and barometric pressure sensors for the effect of changing sensor temperature on sensor bias.
 
 This topic details the [test environment](#test_setup) and [calibration procedures](#calibration_procedures).
 At the end there is a description of the [implementation](#implementation).
 
 :::note
-After thermal calibration the thermal calibration parameters (`TC_*`) are used for *all* calibration/compensation of the respective sensors.
+After thermal calibration, the thermal calibration parameters (`TC_*`) are used for *all* calibration/compensation of the respective sensors.
 Any subsequent standard calibration will therefore update `TC_*` parameters and not the "normal"  `SYS_CAL_*` calibration parameters (and in some cases these parameters may be reset).
 :::
  
 :::note
-At time of writing (PX4 v1.11) thermal calibration of the magnetometer is not yet supported.
+Releases up to PX4 v1.14, do not support thermal calibration of the magnetometer.
 :::
 
 <span id="test_setup"></span>
 ## Test Setup/Best Practice
 
-The [calibration procedures](#calibration_procedures) described in the following sections are ideally run in an *environment chamber* (a temperature and humidity controlled environment) as the board is heated from the lowest to the highest operating/calibration temperature.
+The [calibration procedures](#calibration_procedures) described in the following sections are ideally run in an *environmental chamber* (a temperature and humidity controlled environment) as the board is heated from the lowest to the highest operating/calibration temperature.
 Before starting the calibration, the board is first *cold soaked* (cooled to the minimum temperature and allowed to reach equilibrium).
 
+:::note
+Active electric heating elements will affect the magnetometer calibration values.
+Care should be taken to ensure heating elements are either inactive or sufficiently
+far from the sensor to avoid injecting noice into the magnetometer calibration.
```

Comment:
Noise typo.
```suggestion
Ensure that heating elements are either inactive or sufficiently far from the sensor to avoid injecting noise into the magnetometer calibration.
```